src/libraries/Microsoft.Extensions.Configuration/src/ConfigurationBuilder.cs

```diff
@@ -22,6 +22,14 @@ public class ConfigurationBuilder : IConfigurationBuilder
         /// </summary>
         public IDictionary<string, object> Properties { get; } = new Dictionary<string, object>();
 
+        /// <summary>
+        /// True to call providers GetChildKeys independently.
+        /// </summary>
+#if NET7_0_OR_GREATER
+        public bool CollectChildKeysIndependently { get; init; }
+#else
+        public bool CollectChildKeysIndependently { get; set; }
+#endif
```

Comment:
This is an incompatible API change between TFMs. This isn't something we allow.

Reply:
Reverted